spec/spec_helper.rb

```diff
@@ -25,13 +25,13 @@
           .metadata[:full_description]
           .split(/\s+/, 2).join('/')
           .gsub(/[^\w\/]+/, '_')
-    VCR.use_cassette(name) { example.call }
+    options = example.metadata.slice(:record, :match_on_requests_on)
```

Comment:
Hey @drej2k 

I am seeing some issues with this line (I think because Hash#slice is an ActiveSupport thing). I wonder if this would be better written as:
```
valid_keys = %w[record match_on_requests_on]
options = example.metadata.select { |key,_| valid_keys.include? key }
```

What do you think?

As a side note, I'm hoping to remove ActiveSupport from the specs at some point.

Reply:
Hi Dan,

Brilliant suggestion. I replaced the line with the code you suggested. The gem 'shoulda' is bringing in the dependency of activesupport. As an exercise I removed gem 'shoulda' and the tests still pass ( i didnt think we are using it).

Did you want me to get rid of the gem shoulda and push the gem removal and your code suggestion to this PR?

Reply:
Nice one!

Yeah happy for you to remove `shoulda` as part of this PR :)

Cheers,
Dan.

Reply:
Ok brilliant, I pushed up the changes

Reply:
Ace, thanks for that. I will get this merged in tonight once I check out the tests locally (since Travis doesn't allow private credentials in a PR).

Appreciate your work on this so far, and I'll let you know how I go tonight .

Reply:
Sounds great. Thanks for reviewing and your suggestions!